### ISSUE TITLE
cobblestone is getting pulverized to sand instead of stone dust

### ISSUE THREAD

**Issue by NgoKimPhu** [programgames/Gregblock-revamped#7]
Cobblestones have 2 pulverization recipes to be turned into 1. sand and 2. stone dust, while already having a forge hammer recipe to be turned into sand.

**Comment by programgames** [comment]
Fixed in latest release : https://github.com/programgames/Gregblock-revamped/releases/tag/2.1.1

I removed the both pulverization recipes due to a limited crafttweaker integration for gregtech which was forcing me to delete the first craft ( cobble to sand ) in the macerator.

[proof](https://gyazo.com/421cdbf10a0ca5802acc1af333301d69)

You can simply re-download the pack or just replace this script/config file : 

[gregtech.cfg](https://github.com/programgames/Gregblock-revamped/blob/0eb2b277fc4f72f16e04ad49c32e4af20ad41054/overrides/config/gregtech.cfg)

[gt_machines.zs](https://github.com/programgames/Gregblock-revamped/blob/0eb2b277fc4f72f16e04ad49c32e4af20ad41054/overrides/scripts/gt_machines.zs)

**Comment by NgoKimPhu** [comment]
You can either remove both recipes then add back the stone dust one, or use this piece of script which I already tested (took me a bit of fidgetting - no prior experience with crafttweaker...):
```
for recipe in macerator.recipes {
  for input in recipe.inputs {
    if (input.matches(<minecraft:cobblestone>)) {
      for output in recipe.outputs {
        if (output.name == "tile.sand.default") {
          recipe.remove();
        }
      }
    }
  }
}
```
Could also be improved by checking size = 1 and compare directly the first item instead of doing for loops like that, but I could not find documentation for zenscript anywhere.

**Comment by CrafterKolyan** [comment]
This was fixed by 0eb2b277fc4f72f16e04ad49c32e4af20ad41054
